initializers/sequelize.js

```diff
@@ -99,3 +118,62 @@ module.exports = {
     });
   }
 };
+
+function migrateSequelizeMeta(api, umzug) {
```

Comment:
I think having it fail and tell you to run `db:migrate:old_schema` is a good solution. It also ensures that we're not re-implementing the meta-migration logic from sequelize-cli.